docker/anet-mssql-linux/Dockerfile

```diff
@@ -13,12 +13,14 @@ RUN export DEBIAN_FRONTEND=noninteractive ACCEPT_EULA=Y && \
     apt-get update && \
     apt-get install -y mssql-server=14.0.3006.16-3 mssql-server-fts=14.0.3006.16-3 mssql-tools unixodbc-dev locales sudo && \
     rm -rf /var/lib/apt/lists/* && \
-    locale-gen en_US.UTF-8
-    
+    locale-gen en_US.UTF-8   && \
+    curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash - && \
+    sudo apt-get install -y nodejs && \
+    npm install -g fake-smtp-server    
```

Comment:
Why add this to the MS SQL Docker image? Why not use a readily available Docker image with just fake-smtp-server from the registry (e.g. https://hub.docker.com/r/devoto13/fake-smtp-server )?

Reply:
Ha, great. I didn't know there was a Docker image with the fake-smtp-server. This makes it easier to integrate and there is indeed no need to alter the MS SQL Docker image. I will use it.